cpp_utils.py

```diff
@@ -326,7 +330,10 @@ def post_func_name_change(new_name, ea):
     xrefs = filter(lambda x: x.type == ida_xref.dr_I and x.user == 1, xrefs)
     args_list = []
     for xref in xrefs:
-        member, old_name, struct = ida_struct.get_member_by_id(xref.frm)
+        member = ida_typeinf.tinfo_t().get_udm_bytid(xref.frm)
+        struct = None
+        if member:
+            struct = member.get_parent_struc()
```

Comment:
I think that
```suggestion
            member = ida_typeinf.udm_t()
            ida_typeinf.tinfo_t().get_udm_by_tid(member, xref.frm)
            struct = ida_typeinf.tinfo_t(tid=xref.frm)
```
is more correct, as there is no such a function named `get_parent_struc`

Reply:
ack